src/components/Person/Person.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
+
+function partner(sex, partnerName) {
+  if (!partnerName) {
+    return 'I am not married';
+  }
+
+  if (sex === 'f') {
+    return `My husband's name is ${partnerName}`;
+  }
+
+  return `My wife's name is ${partnerName}`;
+}
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+    <p className="Person__age">
+      {`I am ${age}` || ''}
+    </p>
+
+    <p className="Person__partner">
+      {partner(sex, partnerName)}
+    </p>
+  </section>
+);
+
+Person.defaultProps = {
+  name: undefined,
+  age: undefined,
+  sex: undefined,
+  partnerName: undefined,
```

Comment:
such default properties don't have any sense, unpassed properties will be undefined by default, no need to write it explicitly. Use another default values for these properties.
Also, you have some properties that should exist for sure, so use .isRequired at propTypes